packages/contracts/src/AdminSetup.sol

```diff
@@ -35,14 +39,17 @@ contract AdminSetup is PluginSetup {
         bytes calldata _data
     ) external returns (address plugin, PreparedSetupData memory preparedSetupData) {
         // Decode `_data` to extract the params needed for cloning and initializing the `Admin` plugin.
-        address admin = abi.decode(_data, (address));
+        (address admin, PluginCloneable.TargetConfig memory targetConfig) = abi.decode(
+            _data,
+            (address, PluginCloneable.TargetConfig)
+        );
```

Comment:
Same as with multisig. Let's encode and decode data using a standard mechanism:
- https://github.com/aragon/taiko-contracts/blob/main/src/setup/MultisigPluginSetup.sol#L30C13-L30C41
- https://github.com/aragon/taiko-contracts/blob/main/src/setup/MultisigPluginSetup.sol#L101-L117